components/_details.css

```
@@ -0,0 +1,13 @@
+.details__interessen {
```

I'd keep it in English
```suggestion
.details__interests {
```